### PR TITLE
feat(contact): added driver interface layer and testing driver

### PR DESCRIPTION
**Pull request by Nolan-Arnold**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/graycoreio/daffodil/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Fixes: N/A

## Does this PR introduce a breaking change?
```
[ ] Yes
[ ] No
```

## Other information

### CODE REVIEW

File: libs/contact/src/index.ts [hunk 1]
```diff
@@ -1 +1,2 @@
-export {}
+export { DaffContactServiceInterface } from './driver/interfaces/contact-service.interface'
+export { DaffContactUnion } from './models/contact-union';
```
Comment (lderrickable) [inline libs/contact/src/index.ts]: You'll need to export the token at some point too.

File: libs/contact/tsconfig.spec.json [hunk 2]
```diff
@@ -7,6 +7,7 @@
       "node"
     ]
   },
+  "emitDecoratorMetadata": true,
```
Comment (lderrickable) [inline libs/contact/tsconfig.spec.json]: don't think you need this since it exists in the tsconfig file that this file extends

Comment (Nolan-Arnold) [reply libs/contact/tsconfig.spec.json]: 8e2c34ad64c8c3ed5ce65740228937021fab22d0

File: libs/contact/src/index.ts (hunk 1 above)
Comment (Nolan-Arnold) [reply libs/contact/src/index.ts]: 8e2c34ad64c8c3ed5ce65740228937021fab22d0

**Pull request merged**